### ISSUE TITLE
bug: 

### ISSUE THREAD

**Issue by DavidLRowe** [keymanapp/keyman#4882]
**Describe the bug**
&CasedKeys doesn't work with keys that haven't been defined and therefore produce the US English default. (Perhaps this is how it is designed, in which case this should be a feature request!)

**To Reproduce**
I encountered this when reviewing https://github.com/keymanapp/keyboards/pull/1519 and suggesting the author add: `store(&CasedKeys) 'a'..'b' 'd'..'g' 'i'..'p' 'r'..'w' 'y'..'z'` to the keyboard to apply CapsLock logic to keys that were not redefined. When I added the statement, I found I needed to add: `+ 'a' > 'a'` and `+ 'A' > 'A'` to get CapsLock to work with the "a" key.

**Expected behavior**
I expected that putting 'a' in the &CasedKeys store would apply the CapsLock logic to the existing default rules for 'a' and 'A'.

---

**Keyman for Windows:**
 - OS: Windows 10 20H2 x64
 - Keyman version: 14.0.271

**Keyman Developer:**
 - OS: Windows 10 20H2 x64
 - Keyman Developer version: 14.0.271.0

**Comment by mcdurdin** [comment]
Yes, you are right, this would be 'by design' for now -- it doesn't impact keys that are fall through to the default keyboard. But it's certainly a good point. I'll log this as a feature request for a future update :)

**Comment by DavidLRowe** [comment]
Additional comment: It seems that Keyman Developer's built-in debugger *does* give CapsLock behaviour for non-specified keys.

**Comment by mcdurdin** [comment]
Keyman Developer's built in debugger is getting some long needed TLC in 15.0. Hopefully its behaviour will be much much closer to Keyman's in all cases in this version!

**Comment by darcywong00** [comment]
Alas, we won't be getting to this soon. So we'll just document the limitation on help.keyman for now.
